ui/main/src/app/modules/share/archives-logging-filters/archives-logging-filters.component.ts

```diff
@@ -174,7 +180,13 @@ export class ArchivesLoggingFiltersComponent implements OnInit, OnDestroy, After
         this.processesWithoutProcessGroupMultiSelectOptions = [];
         this.processMultiSelectOptionsPerProcessGroups.clear();
         this.processGroupMultiSelectOptions = [];
+    }
+
+    loadValuesforTags() {
         this.tagsMultiSelectOptions = [];
+        if (this.tags) {
+            this.tags.forEach((tag) => this.tagsMultiSelectOptions.push(new MultiSelectOption(tag.value, tag.label)));
+        }
```

Comment:
same as previously , i did not test it but i guess you need to call the changeDetector